apps/scan/frontend/src/app.test.tsx

```diff
@@ -786,7 +788,10 @@ test('poll worker can open, pause, unpause, and close poll without scanning any
   apiMock.expectSetPollsState('polls_paused');
   apiMock.expectGetConfig({ pollsState: 'polls_paused' });
   userEvent.click(await screen.findByText('Pause Voting'));
-  await screen.findByText('Pausing Voting…');
+  // avoid `act` warning due state changes after async report printing runs
```

Comment:
I thought this warning got removed in React 18?

Reply:
The warning that got removed is "a state update was made to an unmounted component", the "act" updates are alive and well. You can actually turn them off I think if you just don't care, it's just a testing environment thing.